gui/default/vendor/webauthn-json-2.1.1/LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2019 GitHub, Inc.
```

Comment:
According to (the now archived) https://github.com/github/webauthn-json/blob/main/README.md :

___
As of March 2025, stable versions of all major browsers now support the following methods:

[PublicKeyCredential.parseCreationOptionsFromJSON(…)](https://developer.mozilla.org/docs/Web/API/PublicKeyCredential/parseCreationOptionsFromJSON_static)
[PublicKeyCredential.parseRequestOptionsFromJSON(…)](https://developer.mozilla.org/docs/Web/API/PublicKeyCredential/parseCreationOptionsFromJSON_static)
[PublicKeyCredential → .toJSON()](https://developer.mozilla.org/docs/Web/API/PublicKeyCredential)
These should be used instead of @github/webauthn-json.
___
Should we consider this change prior to release?

Reply:
Definitely sounds like a simplification worth adding.